data/core/chitchat.md

```diff
@@ -834,6 +843,28 @@
     - action_chitchat
     - utter_direct_to_step2
 
+## new to rasa/bots, explain core and try out stack
+* greet
+    - action_greet_user
+* how_to_get_started
+    - utter_getstarted
+    - utter_first_bot_with_rasa
+* affirm
+    - action_set_onboarding
+    - slot{"onboarding": true}
+    - utter_built_bot_before
+* deny
+    - utter_explain_stack
+    - utter_stack_details
+    - utter_explain_nlucore
+* how_to_get_started{"product": "core"}
+    - utter_explain_core
+    - utter_also_explain_nlu
+* explain
```

Comment:
not sure this quite makes sense. If the user says "please explain" they could be referring to NLU

Reply:
good point, will change the story